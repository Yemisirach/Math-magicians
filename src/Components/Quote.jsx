import React from 'react';

export default function Quote() {
  return (
    <div>
      <div id="HomeContainer">
        <h3 className="quote-text">
          Mathematics is not about numbers, equation, computations
          <br />
          or algorithms. It is about understandin. - William paul Thurston
        </h3>
      </div>
    </div>
  );
}
