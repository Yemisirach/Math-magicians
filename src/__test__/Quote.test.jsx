import React from "react";
import renderer from "react-test-renderer";
import Quote from "../components/Quote";

describe("Calculator component", () => {
  it("should render correctly and match the saved snapshot", () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
