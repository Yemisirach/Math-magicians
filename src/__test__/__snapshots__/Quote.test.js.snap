// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator component should render correctly and match the saved snapshot 1`] = `
 <div>
      <div id="HomeContainer">
        <h3 className="quote-text">
          Mathematics is not about numbers, equation, computations
          <br />
          or algorithms. It is about understandin. - William paul Thurston
        </h3>
      </div>
    </div>
`;
